src/components/auth/SocialAuthButton.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import SocialAuthButtons from '../presentation/socialAuth';
+
+const { REACT_APP_GOOGLE_LOGIN, REACT_APP_FACEBOOK_LOGIN, REACT_APP_TWITTER_LOGIN } = process.env;
```

Comment:
hi @okoroemeka I can see you're abstracting the URLs as environmental variables, albeit you didn't the env.sample modifications for us to know the URLs to add as environment variables

Reply:
Resolved.

Reply:
👍